webapps/frontend/app/views/site/backend/backend-alba-detail-registered-node.html

```diff
@@ -264,6 +304,11 @@
                     <!-- /ko -->
                 </div>
                 <!-- /ko -->
+            <!-- ko if: !['', null, undefined].contains(emptySlotMessage()) -->
+            <div class="notification-message">
+                <span data-bind="text: 'Could not generate a new slot: ' + emptySlotMessage()"></span>
```

Comment:
We don't allow actual text in our HTML, should be translated in alba.json
Also, the only message that can appear in `emptySlotMessage` is: 'Unable to request an empty slot'. Isn't this message enough to show? Or is this pre-text necessary?